modules/pixi/styles.js

```diff
@@ -1,5 +1,9 @@
 import * as PIXI from 'pixi.js';
 import { osmPavedTags } from '../osm/tags';
+import { ColorSystem } from '../core/ColorSystem';
+
+const instance = new ColorSystem();
```

Comment:
We don't want a new() color system actually- we already did the hard work of starting and initializing the color system inside context.js. What we should instead do is look for `context.systems` and get the color system out of that data structure. There are examples of how to do that all over the code.